data/types/dosingdecision/carbs_on_board.go

```diff
@@ -0,0 +1,55 @@
+package dosingdecision
+
+import (
+	"time"
+
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	MinCarbsOnBoard = 0
+	MaxCarbsOnBoard = 1000
```

Comment:
Although 1000g of carbs on board seems like a sensible maximum for most humans, I'm wondering if it's still possible to exceed. I'm thinking body builders on extreme diets. Even then, 1000g is probably fine.

Reply:
I think the idea is that we go with a number that no one would ever do.  I will increase to 10000